network-api/networkapi/campaign/views.py

```diff
@@ -89,8 +89,8 @@ def signup_submission(request, signup):
         "format": "html",
         "source_url": request.data['source'],
         "newsletters": signup.newsletter,
-        "lang": request.data.get('lang', 'en'),
-        "country": request.data.get('country', 'US'),
+        "lang": request.data['lang'],
```

Comment:
let's file a general bug-fix issue here: if there are arguments that _need_ to be supplied, then we should make sure to check that they exist in the request, before forming a payload, and respond with an appropriate HTTP error code for missing parameters.

Reply:
Alternatively, let's tap into `lang` but fall back to the language wagtail sees in the request, or if there is none, fall back to `en`

Reply:
_Update_: decided to go with original method since pulling wagtail lang data takes you down a rabbit hole 🐇